Library/Homebrew/cmd/install.rb

```diff
@@ -91,6 +91,14 @@ def install
         puts_columns(search_formulae(query))
         puts "Searching taps..."
         puts_columns(search_taps(query))
+
+        if HOMEBREW_REPOSITORY.exist? &&
+           (Time.now.strftime("%s").to_i - `stat -f "%m" #{HOMEBREW_REPOSITORY}/.git/refs/heads/master`.to_i) > 172800
```

Comment:
`HOMEBREW_REPOSITORY` will always exist (it's not the `.git` directory). Also, you can do the stat call in ruby.

```rb
master = HOMEBREW_REPOSITORY.join(".git", "refs", "heads", "master")
if master.exist? && (Time.now.to_i - File.mtime(master).to_i) > 172800
  # ...
end
```